templates/zerver/help/about-streams-and-topics.md

```diff
@@ -4,22 +4,22 @@ In Zulip, conversations are organized by conversation **streams** and
 **topics**.
 
 ## About streams
-On Zulip, users communicate with each other in group chats by sending messages
-to streams, which are similar to conversation threads.
+On Zulip, users communicate with each other in group chats by sending
+messages to streams, which are similar to conversation threads.
 
 Streams are either:
 
-* **Public** - Public streams are for open discussions. All users can subscribe
-to public streams and discuss topics there. Any Zulip user can join any public
-stream in the realm, and they can view the complete message history of any
-public stream without joining the stream.
-* **Private** - Private streams are for confidential discussions and are only
-visible to users who've been invited to subscribe to them. Users who are not
-members of a private stream cannot subscribe to the stream, and they also cannot
-read or send messages to the stream.
-
-Users are subscribed to specific streams in the realm by default, such as the
-[#announce](the-announce-stream) stream. Users can easily
+* **Public** - Public streams are for open discussions. All users can
+subscribe to public streams and discuss topics there. Any Zulip user can
+join any public stream in the realm, and they can view the complete
```

Comment:
* All users can subscribe to public streams
* Any Zulip user can join any public stream in the realm

Both refer to the same thing right? You can remove one.